src/components/tree/tree-node/tree-node.ts

```diff
@@ -153,6 +153,21 @@ export class MTreeNode extends ModulVue {
         }
     }
 
+    private isParentOfOpenedFolder(currentNode: TreeNode = this.node, found: boolean | undefined = undefined): boolean | undefined {
```

Comment:
Idéalement on devrait pas passé le parmètre found en paramètre non?


`private isParentOfOpenedFolder(currentNode: TreeNode = this.node): boolean {
let found =false;
`

qu'est que tu en pense

Reply:
Oops, vestige de mon idée initiale pour la fonction. C'est changé!